packages/react/ds/src/combo-box/dropdown-item.tsx

```diff
@@ -68,7 +68,7 @@ export const DropdownItem = ({
   };
 
   return (
-    <div>
+    <div role="group" aria-label={`${children} dropdown`}>
```

Comment:
what about to expose aria attributes so the user can decide the label?

something like:
```aria-label={props?.aria?.label ||  `${children} dropdown`}```